airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/IntegrationRunner.java

```diff
@@ -92,6 +94,7 @@ public IntegrationRunner(final Source source) {
     validator = jsonSchemaValidator;
   }
 
+  @Trace(operationName = "runOperation")
```

Comment:
Nit:  the other operations use lowercase dash format.  Consider changing this one to be consistent with the others (or change them to be consistent with this lower camel case).